reformcloud2-runner/pom.xml

```diff
@@ -49,7 +49,8 @@
                 <configuration>
                     <archive>
                         <manifestEntries>
-                            <Premain-Class>systems.reformcloud.reformcloud2.runner.instrumentation.RunnerPremain</Premain-Class>
+                            <Premain-Class>systems.reformcloud.reformcloud2.runner.instrumentation.RunnerPremain
+                            </Premain-Class>
```

Comment:
Moving this to the second line definitely causes issues during the maven build process

Reply:
Moved it back.